packages/rum-core/src/domain/view/viewMetrics/trackLargestContentfulPaint.ts

```diff
@@ -61,6 +72,22 @@ export function trackLargestContentfulPaint(
           lcpTargetSelector = getSelectorFromElement(lcpEntry.element, configuration.actionNameAttribute)
         }
 
+        if (
+          !zeroLcpReported &&
+          lcpEntry.startTime === 0 &&
+          isExperimentalFeatureEnabled(ExperimentalFeature.ZERO_LCP_TELEMETRY)
+        ) {
+          zeroLcpReported = true
+          addTelemetryDebug('LCP with startTime = 0', {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I wonder if it could make sense to log whether an anterior LCP entry was notified with a non 0 startTime

Reply:
Yes, it could be nice to know. I added it